src/screens/ExplainSheet.js

```diff
@@ -257,6 +257,39 @@ export const explainers = (params, colors) => ({
     text: VERIFIED_EXPLAINER,
     title: lang.t('explain.verified.title'),
   },
+  unverified: {
+    extraHeight: android ? 150 : 120,
```

Comment:
following the pictures above, feels like the height in android is too long, action buttons margin with bottom is higher than ios and other explain sheets